diff -Nur Curses-UI-0.96/t/99template.t Curses-UI-0.96.mod/t/99template.t
--- Curses-UI-0.96/t/99template.t	2007-12-10 15:49:23.000000000 +0100
+++ Curses-UI-0.96.mod/t/99template.t	2008-09-08 20:35:10.000000000 +0200
@@ -5,8 +5,6 @@
 use lib "$FindBin::RealBin/fakelib";
 use lib "$FindBin::RealBin/../lib";
 
-#### REMOVE ####
-exit 1;
 
 $ENV{LINES} = 25;
 $ENV{COLUMNS} = 80;
@@ -17,12 +15,12 @@
 
 $cui->leave_curses();
 
-isa_ok($cui, "Curses::UI");
+#isa_ok($cui, "Curses::UI");
 
-my $mainw = $cui->add("testw","Window");
+#my $mainw = $cui->add("testw","Window");
 
-isa_ok($mainw, "Curses::UI::Window");
+#isa_ok($mainw, "Curses::UI::Window");
 
-my $wid = $mainw->add("testwidget","<:WIDGET:>");
+#my $wid = $mainw->add("testwidget","<:WIDGET:>");
 
-isa_ok($wid, "<:CLASS:>");
+#isa_ok($wid, "<:CLASS:>");
